mariamihaelafainarea/assignment4/DijointSet

```diff
@@ -0,0 +1,70 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment4;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author maria
+ */
+public class DijointSet {
```

Comment:
Can you document what this class is responsible for and roughly how it works? It's not that easy to follow without any context.

Reply:
Yes. Done it